source/fem-dem/cfd_dem_coupling.cc

```diff
@@ -1248,6 +1248,63 @@ CFDDEMSolver<dim>::postprocess_fd(bool first_iteration)
     }
 }
 
+template <int dim>
+void
+CFDDEMSolver<dim>::postprocess_cfd_dem()
+{
+  // Calculate total volume of fluid and solid 
+  if (this->simulation_parameters.post_processing.calculate_volume_phases)
+   {
+      TimerOutput::Scope t(this->computing_timer, "total_volume_calculation");
+      double             total_volume_fluid, total_volume_particles;
+      std::tie(total_volume_fluid, total_volume_particles) = calculate_fluid_and_particle_volumes(
+        this->void_fraction_dof_handler,
+        this->nodal_void_fraction_relevant,
+        *this->cell_quadrature,
+        *this->mapping);
+      this->table_volume_phases.add_value(
+        "time", this->simulation_control->get_current_time());
+      this->table_volume_phases.add_value("total-volume-fluid", total_volume_fluid);
+      this->table_volume_phases.add_value("total-volume-particles",
+                                          total_volume_particles);
+      if (this->simulation_parameters.post_processing.verbosity ==
+          Parameters::Verbosity::verbose)
+        {
+          this->pcout << "Total volume of fluid: "
+                      << std::setprecision(
+                           this->simulation_control->get_log_precision())
+                      << this->simulation_parameters.physical_properties_manager
+                             .get_density_scale() *
+                           total_volume_fluid
+                      << " m^3" << std::endl;
+          this->pcout << "Total volume of particles: "
+                      << std::setprecision(
+                           this->simulation_control->get_log_precision())
+                      << this->simulation_parameters.physical_properties_manager
+                             .get_density_scale() *
+                           total_volume_particles
+                      << " m^3" << std::endl;
```

Comment:
I was thinking about Inserting the announce string like "CFD-DEM Postprocessing" before starting this function, but now we don't have a announce of "Postprocessing" in the first place, like below.
----
DEM
----
DEM contact search at dem step 0
Finished 100 DEM iterations 
---------------------------------------------------------------
Pressure drop: 1645.91 Pa
Total pressure drop: 1658.26 Pa
Total volume of fluid: 0.0456071 m^3
Total volume of particles: 0.00654498 m^3
---------------
Fluid Dynamics
---------------

I think there are three options here;

- Don't announce anything(Don't change anything)
- Announce only "Post-Process" before starting cfd-postprocessing
- Announce both before cfd-postprocessing and before cfd-dem postprocessing

It might be a big issue, but I just wanna make sure which option looks good for you guys